providers/catalog.go

```diff
@@ -26,6 +26,7 @@ const (
 	DocuSign          Provider = "docuSign"
 	DocuSignDeveloper Provider = "docuSignDeveloper"
 	Calendly          Provider = "calendly"
+	AWeber            Provider = "aweber"
```

Comment:
Even though it's weird, this should be `aWeber`

Reply:
changes this as well.